.github/workflows/spread-tests.yaml

```diff
@@ -228,7 +226,12 @@ jobs:
           # propagated; and we use a subshell as this option could trigger
           # undesired changes elsewhere
           echo "Running command: $SPREAD $RUN_TESTS"
-          (set -o pipefail; $SPREAD -no-debug-output -logs spread-logs $RUN_TESTS | PYTHONDONTWRITEBYTECODE=1 ./tests/lib/external/snapd-testing-tools/utils/log-filter $FILTER_PARAMS | tee spread.log)
+          (
+            set -o pipefail 
+            $SPREAD -no-debug-output -logs spread-logs $RUN_TESTS | \
+            ./tests/lib/external/snapd-testing-tools/utils/log-filter $FILTER_PARAMS | \
```

Comment:
Nitpick, can you ident lines following trailing backslash on the previous line so that it's visually distinct. Thanks!